### PR TITLE
fix(routes): apply the suffix adjustment in batch_action_path

### PR DESCRIPTION
**Pull request by watsy0007**

### CODE REVIEW

**Comment by watsy0007** [comment]
#4781 
thanks @knu

File: spec/unit/resource/routes_spec.rb [hunk 1]
```diff
@@ -41,11 +41,18 @@ module ActiveAdmin
       context "when registering a plural resource" do
         class ::News; def self.has_many(*); end end
         let!(:config) { ActiveAdmin.register News }
-        before{ reload_routes! }
+        before do
+          config.batch_actions= true
```
Comment (timoschilling) [inline spec/unit/resource/routes_spec.rb]: why is this needed?

File: spec/unit/resource/routes_spec.rb [hunk 2]
```diff
@@ -41,11 +41,18 @@ module ActiveAdmin
...
+          reload_routes!
+        end
 
         it "should return the plural route with _index" do
           expect(config.route_collection_path).to eq "/admin/news"
         end
+
+        it "should rturn the pluar route batch action with _index" do
```
Comment (timoschilling) [inline spec/unit/resource/routes_spec.rb]: typo in return

**Comment by timoschilling** [comment]
This is a double of #4781, or not?

**Comment by watsy0007** [comment]
@timoschilling yes
